CHANGES.md

```diff
@@ -1,5 +1,165 @@
+
+### Fixed bugs that could lead to crashes
+
+- [#6074](https://github.com/gap-system/gap/pull/6074) Fix an obscure crash in the GAP interpreter involving character literals like `'\200'`
+
+### Fixed bugs that could lead to unexpected errors
+
+- [#6033](https://github.com/gap-system/gap/pull/6033) Fix regression introduced in GAP 4.13.1 that could cause coset enumeration to fail in cases were it used to work fine in GAP 4.13.0
+- [#5997](https://github.com/gap-system/gap/pull/5997) Fix error in membership test for rational functions
+- [#5955](https://github.com/gap-system/gap/pull/5955) Fix errors for algebra homomorphisms (infinite recursions and "no method found" errors), in particular concerning homomorphisms that are constructed as inverse mappings
+- [#5935](https://github.com/gap-system/gap/pull/5935) Fix an unexpected error in `IsomorphismGroups`
+- [#5908](https://github.com/gap-system/gap/pull/5908) Fix `Extensions` for nonsolvable group and trivial cohomology
+- [#5907](https://github.com/gap-system/gap/pull/5907) Fix unexpected errors in `recog` and `genss` by improving `GroupWithMemory` when given compressed matrices over small fields
+- [#5903](https://github.com/gap-system/gap/pull/5903) Fix `FactorCosetAction(G, G)` for perm. groups
+
+### Other fixed bugs
+
+- [#6070](https://github.com/gap-system/gap/pull/6070) Ensure `InterpolatedPolynomial` always returns a polynomial, even for "constant" input
+- [#6038](https://github.com/gap-system/gap/pull/6038) Fix an unexpected error when using `DotFileLattice`
+- [#6005](https://github.com/gap-system/gap/pull/6005) Ensure ˋDoubleCosets( G, U, V )ˋ checks that both ˋUˋ and ˋVˋ are contained in ˋGˋ, and immediately throws an error if this is not the case
+- [#5936](https://github.com/gap-system/gap/pull/5936) Fix `--quiet` in the command line interface to be identical to `-q`
+- [#5900](https://github.com/gap-system/gap/pull/5900) Fix `IsomorphismPermGroupOrFailFpGroup` to honor its second argument, which limits the coset table size that gets used before it gives up
+- [#5864](https://github.com/gap-system/gap/pull/5864) Fix 'gap --version' to send its output to stdout and include a trailing newline
+
+### Other changes
+
+- [#5904](https://github.com/gap-system/gap/pull/5904) Change three argument version of `Assert` to raise an error if the assertion condition evaluates to `false` (instead of merely printing a message)
+
+### Package distribution
+
+#### New packages redistributed with GAP
+
+- [**ALCO**](https://bnasmith.github.io/alco/) 1.1.1: Tools for algebraic combinatorics, by Benjamin Nasmith
+- [**IBNP**](https://gap-packages.github.io/ibnp/) 0.16: Involutive Bases for Noncommutative Polynomials, by Gareth A. Evans, Christopher D. Wensley
+- [**InduceReduce**](https://gap-packages.github.io/InduceReduce/) 1.1: Unger's algorithm to compute character tables of finite groups, by Jonathan Gruber
+
+#### Packages no longer redistributed with GAP
+
+- **ToricVarieties**: TODO
```

Comment:
Is there any more to say about this?

Reply:
Resolved now, would be great if you could double check the text I wrote